README.adoc

```diff
@@ -274,7 +269,7 @@ because the server is configured to allow all origins, and the request came with
 
 Since you started Open Liberty in development mode at the start of the guide, press the `enter/return` key to run the tests.
 
-The [hotspot=testPreflightCorsRequest file=1]`testPreflightCorsRequest` test passes and prints the response headers with their values from the endpoint.
+If the [hotspot=testPreflightCorsRequest file=1]`testPreflightCorsRequest` test passes, it will print the response headers with their values from the endpoint.
```

Comment:
Update

>If the `testPreflightCorsRequest` test passes, it will print the response headers with their values from the endpoint.

To

>If the `testPreflightCorsRequest` test passes, the response headers with their values from the endpoint are printed.

Reply:
Addressed.